.idea/.rakeTasks

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The .idea folder is specific to your text editor (RubyMine). You should add it to your global .gitignore and remove it from this repo.